source/02.04_header.sub

```diff
@@ -104,7 +99,7 @@ as a quick look initial determination of file type.
 
 This field should be set to a value from 0 to 65,535. A value of zero is
 interpreted to mean that an ID has not been assigned, which is the norm for a
-LAS file resulting from an aggregation of multiple independent sources (e.g., a
+file resulting from an aggregation of multiple independent sources (e.g., a
```

Comment:
Why did you remove "LAS" in multiple places? Does it handle a certain use case?

Reply:
This is a specification for LAS. Repeating the acronym "LAS" adds nothing. It's also more consistent.